scope

```diff
@@ -90,8 +90,14 @@ check_not_running() {
 }
 
 launch_command() {
+    local args="-v /etc/weave:/etc/weave"
```

Comment:
Sorry about that!  We put a signature file in there, and mount it into the container to ensure the same signature is sent to the checkpoint server between runs of the scope container.